plugin/build.gradle

```diff
@@ -0,0 +1,15 @@
+dependencies {
+  compileOnly 'com.google.errorprone:error_prone_core:latest.release'
+  compileOnly 'com.google.errorprone:error_prone_annotation:latest.release'
+  testImplementation("com.google.errorprone:error_prone_test_helpers:latest.release")
+
+  compileOnly         'com.google.auto.service:auto-service:1.0-rc4'
+  annotationProcessor 'com.google.auto.service:auto-service:1.0-rc4'
+}
+
+//configurations {
```

Comment:
Are these comments from the Error Prone sample code?

Reply:
At one point, I could not get the tests to build without that part. I commented it out because it was fixed by something else I did. I can keep a note of it somewhere else in case I need it in the future and remove it from the code.